datafusion-examples/examples/dataframe.rs

```diff
@@ -28,16 +28,24 @@ use std::io::Write;
 use std::sync::Arc;
 use tempfile::tempdir;
 
-/// This example demonstrates using DataFusion's DataFrame API to
+/// This example demonstrates using DataFusion's DataFrame API
```

Comment:
Note the `to_date` function isn't even listed in the readme menu: https://github.com/apache/datafusion/blob/main/datafusion-examples/README.md